examples/tests/invalid/map_entries.test.w

```diff
@@ -0,0 +1,2 @@
+let map = { 1 => "foo", 2 => "bar" };
```

Comment:
can we add an invalid test to ensure type safety. something like:
```js
let map = { "foo": 1, "bar": 2 };

let entries = map.entries();

let someStr: str = entries.at(0).value;
```

Reply:
Sure! I'll put it in place!